analysis/mailchimp/mailchimp_gains_losses.sql

```diff
@@ -0,0 +1,47 @@
+with
+email_summary as
+(
+	select *
+	from {{env.schema}}.mailchimp_email_summary
+),
+
+gains as (
+	select date_trunc('month', signup_date) as month, count(*) total_gains
+	from {{env.schema}}.mailchimp_members
+	group by date_trunc('month', signup_date)
+),
+
+losses as (
+	select month, count(*) as total_losses
+	from
+	(
+		-- find everyone who unsubscribed
+	  	select date_trunc('month', unsubscribed_date) as month
+	  	from email_summary
+	  	where unsubscribed_date is not null
+	  	-- add everyone with a hard bounce
+	  	union
+	  	select date_trunc('month', hard_bounced_date) as month
+	  	from email_summary
+	  	where hard_bounced_date is not null
+	)
+	group by month
+),
+
+months as (
```

Comment:
Is there a reason you didn't use the date trick that we've used before in zuora and stripe to make sure we're actually getting all months here? At the current time it's possible that certain months would not be represented here, correct?